src/features/RecipeEdit/components/RecipeForm.tsx

```diff
@@ -95,6 +95,7 @@ const RecipeForm: React.FC<Props> = ({
         reason: AutocompleteChangeReason,
     ) => {
         // One of "createOption", "selectOption", "removeOption", "blur" or "clear".
+        // eslint-disable-next-line
         if (reason === "selectOption" || "createOption" || "removeOption") {
```

Comment:
I believe this is a legit complaint: that conditional makes no sense? I opened #190 to investigate. Unrelated to this PR, so no need to delay.